server.js

```diff
@@ -37,7 +37,8 @@ app.get("/coolify", (request, response) => {
 		cool_name_curr: cool_names[10],
 		cool_name_fairy: cool_names[11],
 		cool_name_dot: cool_names[12],
-    cool_name_doublestruck: cool_names[13],
+    		cool_name_doublestruck: cool_names[13],
```

Comment:
Fix the indentation.

Reply:
Fixed the indentation https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/5729fd65abecf22197242ac91504d633a7f9d7f7